generator/ServiceClientGeneratorLib/Member.cs

```diff
@@ -583,7 +583,8 @@ private string DetermineType(JsonData extendedData, bool treatEnumsAsString, boo
                     var valueType = DetermineType(memberShape["value"], true, false);
                     return string.Format("Dictionary<{0}, {1}>", keyType, valueType);
                 case "list":
-                    var listType = DetermineType(memberShape["member"], true, false);
+                    // if customization exists to treat the enum as an enum instead of a string, we pass in false
+                    var listType = this.model.Customizations.OverrideTreatEnumsAsString(this.Extends) ? DetermineType(memberShape["member"], false, false) : DetermineType(memberShape["member"], true, false) ;
```

Comment:
This `?:` can be reduced down `DetermineType(memberShape["member"], this.model.Customizations.OverrideTreatEnumsAsString(this.Extends), false)`